charts/everest/templates/hooks.yaml

```diff
@@ -1,3 +1,27 @@
 {{- if .Values.upgrade.preflightChecks }}
 {{- include "everest.preUpgradeChecks" (dict "namespace" (include "everest.namespace" .) "version" .Chart.Version "versionMetadataURL" (include "everest.versionMetadataURL" .)) }}
 {{- end }}
+---
+{{- /*
+# --- Pod Scheduling Policies
+# During the installation of Everest v1.7.0 we need to create default pod scheduling policies.
+*/}}
+
+{{- if  .Release.IsInstall }}
+{{- include "everest.createDefaultPsp" . }}
+{{- end }}
+
+{{- /*
+# During the upgrade to Everest v1.7.0 we need to:
+#  - create default pod scheduling policies.
+#  - assign default pod scheduling policies to all existing DB clusters.
+ */}}
+{{- /* if and (.Release.IsUpgrade) (eq .Chart.Version "1.7.0") */}}
```

Comment:
Should this be uncommented?

Reply:
yes, just before we release v1.7.0. Otherwise it is not executed during upgrading v1.6.0 to Feature Build